model/src/main/java/org/cbioportal/genome_nexus/model/Snpeff.java

```diff
@@ -0,0 +1,34 @@
+package org.cbioportal.genome_nexus.model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.List;
+
+public class Snpeff
+{
+    private String license;
+
+    //private List<Ann> ann;
```

Comment:
remove this one?

Reply:
I filed an issue to my variant info because Ann sometimes appears as a dictionary instead of a list. They responded that it was what they intended. Would it be possible to cast the dictionary into a list if it isn't a list in the output of myvariant.info?